kolibri/plugins/device/assets/src/views/FacilitiesPage/FacilityTaskPanel.vue

```diff
@@ -111,12 +109,7 @@
         if (this.taskInfo.canCancel) {
           return 'cancel';
         } else if (this.taskInfo.canClear) {
-          // Import tasks can't be retried since we don't save the username/password
```

Comment:
This comment was incorrect. The credentials are validated prior to the creation of the task, and once validated, a certificate is created which facilitates authentication thereafter